building/building_index.adoc

```diff
@@ -1,26 +1,25 @@
 // vim: set syntax=asciidoc:
-[[build]]
-== Building Applications
+[[Building]]
+== Building Images
 :data-uri:
 :homepage https://github.com/projectatomic/container-best-practices:
 
+=== Simple Build
 
-=== Simple build
-
-For building docker containers, we must first have the docker daemon installed and running:
+For building docker images, we must first have the docker daemon installed and running:
 [source,shell]
 ----
 #> yum install -y docker
```

Comment:
I would prefer to mention dnf/yum. Or use to sections. One for RHEL and the other one for Fedora.

Reply:
I would personally just mention that environment is an enterprise linux (=yum): this is not a guide how to install packages, so we don't need to be precise in this matter. My point is, what if some community contributor sends a PR with installation steps for Ubuntu, or ArchLinux -- would you merge such patch?